cts/scheduler/summary/failcount-block.summary

```diff
@@ -1,3 +1,6 @@
+error: Ignoring invalid node_state entry without id
+warning: Ignoring failure timeout (10s) for rsc_pcmk-2 because it conflicts with on-fail=block
+warning: Ignoring failure timeout (10s) for rsc_pcmk-4 because it conflicts with on-fail=block
```

Comment:
Most of the errors/warnings in the test cases are things we really should fix in the test case :(

(some may be testing the error though)